File: Spring REST API/spring/src/main/java/be/pxl/spring/model/Session.java
package be.pxl.spring.model;

import java.sql.Timestamp;
import org.springframework.context.annotation.Scope;
import org.springframework.stereotype.Component;

@Component
@Scope("prototype")
public class Session {

	int sessionId, userId, voiceTime;
	Timestamp startTime, endTime;
	
}

File: UWP app/UWPMonitoring/UWPMonitoring.Models/Session.cs
﻿using System;

namespace UWPMonitoring.Models
{
    public class Session
    {
        public int SessionId { get; set; }

        public int UserId { get; set; }

        public DateTime Start_Time { get; set; }

        public DateTime End_Time { get; set; }

        public DateTime Actual_Time { get; set; }
    }
}